stable/spark-history-server/templates/deployment.yaml

```diff
@@ -35,7 +35,7 @@ spec:
           value: "true"
         ports:
         - name: historyport
-          containerPort: {{ .Values.service.port }}
+          containerPort: 18080
```

Comment:
Any reason you want to hard code the port?

Reply:
I want to have port 80 on a service, when I set it to the value other than 18080 (HS default) it no longer points to HS Web UI.

Reply:
Got it. Thanks.